apps/designer/app/designer/features/sidebar-left/lib/navigator/navigator.tsx

```diff
@@ -1,11 +1,18 @@
 import { Box } from "@webstudio-is/design-system";
-import type { Instance, Publish } from "@webstudio-is/react-sdk";
+import type { Instance } from "@webstudio-is/react-sdk";
+import { type Publish } from "~/shared/pubsub";
 import { useSelectedInstancePath } from "~/designer/shared/instance/use-selected-instance-path";
 import { useSelectedInstanceData } from "~/designer/shared/nano-states";
 import { Tree } from "~/designer/shared/tree";
 import { useRootInstance } from "~/shared/nano-states";
 import { Header } from "../header";
 
+declare module "~/shared/pubsub" {
```

Comment:
what worries me about these is that their location is sort of random, how do we know where to put those declarations? Should we centralize them for things that are "shared" and not actually local?

Reply:
Ye, I think that putting shared things in the central spot would be better. It just wasn't always obvious to me where things should be put and which ones are local and which ones are not. Those random places in which you can find those augmentations right now are usually in files that publish those events.